boards/nxp/imx8mp_evk/doc/index.rst

```diff
@@ -205,6 +210,87 @@ DDR
    dcache flush
    bootaux 0x80000000
 
+Load and Run M7 Zephyr Image by using Linux remoteproc
+======================================================
+
+Prepare device tree:
+The device tree must inlcude CM7 dts node with compatible string "fsl,imx8mn-cm7",
+and also need to reserve M4 DDR memory if using DDR code and sys address, and also
+need to put "m4_reserved" in the list of memory-region property of the cm7 node.
+
+.. code-block:: console
+
+   reserved-memory {
+            #address-cells = <2>;
+            #size-cells = <2>;
+            ranges;
+
+            m7_reserved: m4@80000000 {
+                  no-map;
+                  reg = <0 0x80000000 0 0x1000000>;
+            };
+            ...
+   }
+
+
+   imx8mp-cm7 {
+            compatible = "fsl,imx8mn-cm7";
+            rsc-da = <0x55000000>;
+            clocks = <&clk IMX8MP_CLK_M7_DIV>,
+                     <&audio_blk_ctrl IMX8MP_CLK_AUDIO_BLK_CTRL_AUDPLL_ROOT>;
+            clock-names = "core", "audio";
+            mbox-names = "tx", "rx", "rxdb";
+            mboxes = <&mu 0 1
+                     &mu 1 1
+                     &mu 3 1>;
+            memory-region = <&vdevbuffer>, <&vdev0vring0>, <&vdev0vring1>, <&rsc_table>, <&m7_reserved>;
+            status = "okay";
+            fsl,startup-delay-ms = <500>;
+   };
+
+Extra Zephyr Kernel configure item for DDR Image:
+
+If use remotepoc to boot DDR board (imx8mp_evk/mimx8ml8/m7/ddr), also need to enable
+"CONFIG_ROMSTART_RELOCATION_ROM" in order to put romstart memory section into ITCM because
+M7 Core will get the first instruction from zero address of ITCM, but romstart relocation
+will make the storage size of zephyr.bin is too large, so we don't enable it by default in
```

Comment:
s/make the storage size of zephyr.bin is too large/make the storage size of zephyr.bin too large
Just remove "is" before "too large"

Reply:
Fixed, thanks.